src/contractTest/java/uk/gov/hmcts/reform/em/annotation/IdamConsumerTest.java

```diff
@@ -207,9 +217,9 @@ public RequestResponsePact executeGetIdamAccessTokenAndGet200WithTimeBoundToken(
 
         Map<String, Object> params = new TreeMap<>(String.CASE_INSENSITIVE_ORDER);
 
-        params.put("redirect_uri", "http://www.service.com/callback");
-        params.put("client_id", "service");
-        params.put("client_secret", "secret");
+        params.put("redirect_uri", redirect_uri);
+        params.put("client_id", client_id);
+        params.put("client_secret", client_secret);
         params.put("scope", "openid roles profile");
         params.put("username", "emCaseOfficer@fake.hmcts.net");
```

Comment:
can you change this user to the one we are creating in the given a user exists i.e, emCaseOfficer@email.net